ext/ddtrace.c

```diff
@@ -392,6 +430,7 @@ static zend_object *ddtrace_span_stack_clone_obj(zend_object *old_obj) {
 static void ddtrace_span_data_free_storage(zend_object *object) {
     zend_object_std_dtor(object);
     // Prevent use after free after zend_objects_store_free_object_storage is called (e.g. preloading) [PHP < 8.1]
+    memset(&object->properties_table[8], 0, sizeof(((ddtrace_span_data *)NULL)->properties_table_placeholder[8])); // links property
```

Comment:
```suggestion
```
`object->properties_table[8]` is pointing to the `parent` property, not the links property. Unless this was necessary, but then there's another bug.